pkg/backend/openshiftcluster.go

```diff
@@ -295,6 +295,7 @@ func (ocb *openShiftClusterBackend) endLease(ctx context.Context, log *logrus.En
 
 func (ocb *openShiftClusterBackend) asyncOperationResultLog(log *logrus.Entry, initialProvisioningState api.ProvisioningState, backendErr error) {
 	log = log.WithFields(logrus.Fields{
+		"logKind":       "asyncqos",
```

Comment:
Could you please link where this tag is going to be used?

Reply:
This tag is used so that when we get to ARO.Pipeline we can use mdsd to create a new table in kusto based off of the fluentBit tag which in this case, it gets from logKind.

Reply:
@rogbas there will be another change in mdsd.xml to route this tag to its own table